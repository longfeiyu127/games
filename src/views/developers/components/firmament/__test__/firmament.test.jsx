import React from 'react'
import renderer from 'react-test-renderer'
import Firmament from '../firmament.tsx'

describe('<Firmament />', () => {
  it('component developers/Firmament render', () => {
    const Dom = renderer.create(<Firmament />).toJSON()
    expect(Dom).toMatchSnapshot()
  })
})
