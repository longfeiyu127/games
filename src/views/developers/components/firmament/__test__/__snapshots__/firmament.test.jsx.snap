// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Firmament /> component developers/Firmament render 1`] = `
<div
  className="c-firmament"
>
  <div
    className="dawn"
  />
  <div
    className="star"
  >
    <div
      className="star-shadow"
    />
  </div>
  <div
    className="multiple-star before"
  />
  <div
    className="multiple-star"
  />
  <div
    className="multiple-star after"
  />
  <div
    className="shooting-star"
  />
  <div
    className="shooting-star after"
  />
</div>
`;
